lib/plugins/Label_83.ts

```diff
@@ -0,0 +1,79 @@
+import { DecoderPlugin } from '../DecoderPlugin';
+import { DecodeResult, Message, Options } from '../DecoderPluginInterface';
+import { CoordinateUtils } from '../utils/coordinate_utils';
+import { ResultFormatter } from '../utils/result_formatter';
+
+export class Label_83 extends DecoderPlugin {
+  name = 'label-83';
+
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ['83'],
+    };
+  }
+
+  decode(message: Message, options: Options = {}) : DecodeResult {
+    const decodeResult = this.defaultResult();
+    decodeResult.decoder.name = this.name;
+    decodeResult.message = message;
+    decodeResult.formatted.description = 'Airline Defined';
+
+    // Inmarsat C-band seems to prefix normal messages with a message number and flight number
+    let text = message.text;
+    if (text.match(/^M\d{2}A\w{6}/)) {
+        ResultFormatter.flightNumber(decodeResult, message.text.substring(4, 10).replace(/0/g, ""));
+        text = text.substring(10);
+    }
+
+    decodeResult.decoded = true;
+    if (text.substring(0, 10) === "4DH3 ETAT2") {
```

Comment:
Might be helpful later if we had a comment here about this match.

Reply:
+1 just a comment linking this to the variant in the tests